lib/stack_master/test_driver/cloud_formation.rb

```diff
@@ -106,7 +108,7 @@ def describe_change_set(options)
       def execute_change_set(options)
         change_set_id = options.fetch(:change_set_name)
         change_set = @change_sets.fetch(change_set_id)
-        update_stack(change_set)
+        @stacks[change_set.fetch(:stack_name)].attributes = change_set
```

Comment:
I would prefer the dead code is removed, as it adds confusion and is easily found in git history.